alembic/latiss/versions/53707815663e_change_primary_key_to_day_obs_seq_num.py

```diff
@@ -0,0 +1,210 @@
+"""change primary key to day_obs + seq_num
+
+Revision ID: 53707815663e
+Revises: 59776480aa4d
+Create Date: 2024-09-11 00:15:14.413772+00:00
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision: str = "53707815663e"
+down_revision: Union[str, None] = "59776480aa4d"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("fk_exposure_id", "ccdexposure", schema="cdb_latiss", type_="foreignkey")
+    op.drop_constraint("fk_obs_id", "exposure_flexdata", schema="cdb_latiss", type_="foreignkey")
+    op.drop_constraint("un_exposure_id_detector", "ccdexposure", schema="cdb_latiss", type_="unique")
```

Comment:
I don't think these have to be dropped.  They're all still valid; even if the first two are not pointing to primary keys, they point to unique keys.